components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/StringArrayResources.kt

```diff
@@ -48,6 +48,20 @@ fun stringArrayResource(resource: StringArrayResource): List<String> {
 suspend fun getStringArray(resource: StringArrayResource): List<String> =
     loadStringArray(resource, DefaultResourceReader, getResourceEnvironment())
 
+/**
+ * Loads a list of strings using the specified string array resource.
+ *
+ * @param environment The resource environment.
+ * @param resource The string array resource to be used.
+ * @return A list of strings representing the items in the string array.
+ *
+ * @throws IllegalStateException if the string array with the given ID is not found.
+ */
+suspend fun getStringArray(
```

Comment:
To be consistent with other types, maybe we need `suspend fun vectorResource(ResourceEnvironment`, `suspend fun imageResource(ResourceEnvironment`, `suspend fun font(ResourceEnvironment`

Reply:
I wouldn't add public APIs without explicit needs.